modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/upgrade-catch-all-check/LPD_52326.testjava

```diff
@@ -0,0 +1,110 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Rubens Soares
+ */
+public class LPD_52326 {
+
+	public void method(
```

Comment:
@rubenssoares-br its a pattern to create the method equals to the method parameters that is being tested. For all those are the same or we could split? If we were to split it up, how many methods would there be?

Reply:
Talk to me in private before doing this part, please.